src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -1048,6 +1072,42 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
                                 IsLocked = methodRoyaltyEntry.IsLocked,
                             });
                         }
+
+                        if (substateData is CoreModel.TypeInfoModuleFieldTypeInfoSubstate typeInfoSubstate)
+                        {
+                            if (typeInfoSubstate.Value.Details is CoreModel.ObjectTypeInfoDetails { InstanceSchema: not null } objectTypeInfoDetails)
```

Comment:
While this is nothing more than personal preference this pattern matching is so difficult to follow... maybe it would be reasonable to introduce `TryGetXxx(out var ...)` extension method on `TypeInfoSubstate`?